hydra_genetics/utils/software_versions.py

```diff
@@ -321,7 +322,7 @@ def export_software_version_as_files(software_dict, directory="versions/software
         output_file_list.append(output_file)
         with open(output_file, 'w') as writer:
             notification = software_dict[name].pop('NOTE', None)
-            writer.write(yaml.dump(software_dict[name]))
```

Comment:
have you checked that you don't get a lot of extra information when you remove name?

Reply:
I ran it on Poppy and it all looks good, so i think it should be fine.

![Screenshot from 2024-05-24 12-10-58](https://github.com/hydra-genetics/hydra-genetics/assets/45485665/589b012d-9d9e-4839-bab1-bd2cacc1e577)